spaces/aliases.json

```
@@ -24,5 +24,6 @@
 	"akropolis-delphi": "akropolis.eth",
 	"rally": "rallygov.eth",
 	"debaseonomics": "debaseonomics.eth",
-  "pickle": "pickle.eth"
+  	"pickle": "pickle.eth",
+	"snowswap": "snowswap.eth"
```

When creating a space you do not have to edit the `aliases.json` file.
```suggestion
  "pickle": "pickle.eth"
```